tests/main/services-start/task.yaml

```
@@ -0,0 +1,49 @@
+summary: Check that "snap start <snap>" affects only enabled services
+
+environment:
+    EXTRA_FLAGS/start: ""
+    EXPECTED_STATUS: "inactive"
```

was this meant to be empty? or did you mean to have it like

```suggestion
    EXPECTED_STATUS/start: "inactive"
```

---

Mmm... it's not clear to me what I was thinking :-) But indeed, adding `/start` makes sense. Done!